doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -195,5 +228,227 @@ set up specifically for issues related to the Weather Model.
 =================
 Running the model
 =================
-The `UFS Weather Model wiki <https://github.com/ufs-community/ufs-weather-model/wiki>`_ includes a simple
-test case that illustrates how the model can be run.
+
+.. _UsingRegressionTest:
+
+--------------------------------
+Using the regression test script
+--------------------------------
+The regression test script ``rt.sh`` in the tests/ directory can be
+used to run a number of preconfigured test cases. It is the top-level script
+that calls lower-level scripts to build, set up environments and run tests.
+On `Tier-1 platforms <https://github.com/ufs-community/ ufs-weather-model/wiki
+/Regression-Test-Policy-for-Weather-Model-Platforms-and-Compilers>`_, it can
+be as simple as editing the ``rt.conf`` file and subsequently executing
+
+.. code-block:: console
+
+    ./rt.sh -l rt.conf
+
+Following discussions are general, but the user may not be able to successfully
+execute the script as is unless s/he is on one of the Tier-1 platforms.
+
+Each line in the PSV (Pipe-separated values) file ``rt.conf`` is used to either
+build or run. The ``COMPILE`` line specifies the application to build (e.g.
+``APP=S2S``), CCPP suite to use (e.g. ``SUITES=FV3_GFS_2017_coupled``), and
```

Comment:
I am thinking maybe it's good to use control as example, as we are in the process of removing the old GFS_2017 suite files for coupled model tests.

Reply:
Good point. I will change it to control.

Reply:
@MinsukJi-NOAA At line 223 in your rst file, it says that the model executable is named ufs_weather_model in the top-level directory. You didn't change this line, so I can't comment directly there, but could you fix it to say the executabel is called ufs_model in the build directory?